torch/_dynamo/variables/constant.py

```diff
@@ -78,6 +82,12 @@ def call_method(
                 items=self.unpack_var_sequence(tx), source=self.source, **options
             ).call_method(tx, name, args, kwargs)
 
+        if any([isinstance(x, DynamicShapeVariable) for x in args]):
+            if name == "__add__":
+                assert len(args) == 1
+                return args[0].call_method(tx, name, [self], {})
+            # Unfortunate constant
```

Comment:
I don't understand this comment

Reply:
This is just a coverage problem right? The super call is just going to raise unimplemented IIUC?

Reply:
This is just the old behavior, it used to always just do `return ConstantVariable(op(self.value, const_args[0]), **options)`